src/promptflow/tests/sdk_cli_test/e2etests/test_flow_run.py

```diff
@@ -1295,7 +1295,9 @@ def test_get_incomplete_run(self, local_client, pf) -> None:
             )
 
             # remove run dag
-            shutil.rmtree(f"{temp_dir}/print_env_var")
+            # TODO 2945229: Han to check whether this is expected - can't delete the whole directory given .promptflow
+            #   is occupied
```

Comment:
why .promptflow is occupied?

Reply:
Also curious with the same thing

Reply:
This is temporary directory like /tmp/xxxx/print_env_var or C:\User\xxx\Temp\print_env_var.
But it is still weird why the folder is not released after pf run.